src/editors/array.js

```diff
@@ -595,13 +596,7 @@ JSONEditor.defaults.editors.array = JSONEditor.AbstractEditor.extend({
 
         self.onChange(true);
 
-        self.jsoneditor.trigger('move');
-
-        // TODO we can remove this eventually
-        self.rows.forEach(function (row) {
-          row.onMove();
-        });
-
+        self.jsoneditor.trigger('moveRow');
```

Comment:
Do we still have `move` events? If so, where?

Reply:
the `row.onMove();`  was the first attempt and was never used in production nor documented. 

s`elf.jsoneditor.trigger('moveRow');` ist already implemented and tested in array editor. It was missing when arrays editors have the table format (table editor).